java/opendcs/src/main/java/decodes/dbeditor/TraceDialog.java

```diff
@@ -88,6 +91,14 @@ public void actionPerformed(ActionEvent e) {
 				closeButton_actionPerformed(e);
 			}
 		});
+		clearButton.setText(
+			genericLabels.getString("clear"));
+		clearButton.addActionListener(new java.awt.event.ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				clear();
+			}
+		});
```

Comment:
Use the shorter lambda syntax here.
```suggestion
		clearButton.addActionListener(e -> clear());
```